op-challenger/cmd/main_test.go

```diff
@@ -462,41 +455,25 @@ func requiredArgs(traceType config.TraceType) map[string]string {
 		"--datadir":              datadir,
 	}
 	switch traceType {
-	case config.TraceTypeAlphabet:
-		addRequiredAlphabetArgs(args)
 	case config.TraceTypeCannon:
 		addRequiredCannonArgs(args)
-	case config.TraceTypeOutputCannon:
-		addRequiredOutputCannonArgs(args)
-	case config.TraceTypeOutputAlphabet:
-		addRequiredOutputAlphabetArgs(args)
+	case config.TraceTypeAlphabet:
+		addRequiredOutputArgs(args)
```

Comment:
Seems like this should this be the following unless we're explicitly trying to say that the rollup rpc arg is needed for outputs?
```suggestion
		addRequiredAlphabetArgs(args)
```

Reply:
The rollup rpc arg is needed for outputs :)

Reply:
Yep cool, it's a little confusing since there's no more reference to outputs but nbd